pdns/pdnsutil.cc

```diff
@@ -1504,6 +1504,23 @@ bool disableDNSSECOnZone(DNSSECKeeper& dk, const DNSName& zone)
   return true;
 }
 
+int setZoneAccount(const DNSName& zone, const string &account)
+{
+  UeberBackend B("default");
+  DomainInfo di;
+  std::vector<std::string> meta;
```

Comment:
It is. A copy/paste from setZoneKind() which seems to have the same small issue. Probably a copy/paste of showZone() where it is actually used. :)